docs/pages/providers/authentik.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Authentik"
+---
+
+# Authentik
+
+For usage, see [OAuth 2.0 provider with PKCE](/guides/oauth2-pkce).
+
+```ts
+import { Authentik } from "arctic";
+
+const AUTHENTIK_URL = "http://localhost:9000";
+
+export const authentik = new Authentik(
+	AUTHENTIK_URL,
+	AUTHENTIK_CLIENT_ID,
+	AUTHENTIK_CLIENT_SECRET,
+	AUTHENTIK_REDIRECT
+);
+```
+
+Authentik with version 2024.2 and higher receive only access token. To get the refresh token as well, you need to include scope `offline_access` in the request. The scope also needs to be enabled in the provider advanced settings.
+
+`Application > Providers > Edit > Advanced protocol settings > Scopes`
+
+```ts
+const scopes = ["profile", "email", "openid", "offline_access"];
+
+const url: URL = await authentik.createAuthorizationURL(state, {
+	scopes
```

Comment:
```suggestion
	scopes: ["profile", "email", "openid", "offline_access"]
```

Reply:
I think that for devs it will be better to have the option to pass the scope manually from envs rather than having them here. I assume you can choose not to include the email or profile. Furthermore if there is any change in the upcoming release or previous releases, you may need different scopes.
But of course I can update this just fine as well if you want. Let me know what you prefer.
Also thanks for the review